.github/workflows/build_frontend.yaml

```diff
@@ -44,6 +44,21 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14'
-      - run: yarn install
-      - run: yarn run build
 
+      - run: |
+          yarn install
+
+      - name: Build release for production branch (conditional check)
+        if: github.ref == 'refs/heads/production'
+        run: |
+          yarn build:release
+
+      - name: Build staging main/master branch (conditional check)
+        if: (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main')
```

Comment:
nice anticipation of master => main

Reply:
Thanks. Someday we can switch. Just wish it wasn't such a high risk renaming.
I figure we can start adding support for both as we touch code.